modules/oe_theme_helper/oe_theme_helper.module

```diff
@@ -29,7 +29,9 @@ function oe_theme_helper_page_attachments(array &$page) {
   }
 
   // Load ECL component library assets.
-  if (\Drupal::service('theme.manager')->getActiveTheme()->getName() === 'oe_theme') {
+  $active_theme = \Drupal::service('theme.manager')->getActiveTheme();
+  // Loading libraries when active theme is oe_theme or oe_theme's child.
+  if ($active_theme->getName() === 'oe_theme' || array_key_exists('oe_theme', $active_theme->getBaseThemes())) {
```

Comment:
Usage of \Drupal\Core\Theme\ActiveTheme::getBaseThemeExtensions() is strongly preferred because getBaseThemes() is deprecated in D8.7.
```suggestion
  if ($active_theme->getName() === 'oe_theme' || array_key_exists('oe_theme', array_keys($active_theme->getBaseThemeExtensions()))) {
```